docs/app/Examples/modules/Progress/Content/index.js

```diff
@@ -17,7 +17,10 @@ const ProgressContentExamples = () => (
     <ComponentExample
       title='Label'
       description='A progress element can contain a label.'
-      examplePath='modules/Progress/Content/ProgressExampleLabel'
+      examplePath='modules/Progress/Content/ProgressExampleLabelPercent'
+    />
+    <ComponentExample
+      examplePath='modules/Progress/Content/ProgressExampleLabelRatio'
```

Comment:
Let's add descriptions:
- for `LabelPercent`: A progress element can contain a percent label.
- for `LabelRatio`: A progress element can contain a ratio label.